blocks/library/pullquote/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlock, query as hpq } from 'api';
+import Editable from 'components/editable';
+
+const { children, query } = hpq;
+
+registerBlock( 'core/pullquote', {
+	title: wp.i18n.__( 'Pullquote' ),
+	icon: 'format-quote',
+	category: 'common',
+
+	attributes: {
+		value: query( 'blockquote > p', children() ),
+		citation: children( 'footer' ),
+	},
+
+	edit( { attributes, setAttributes, focus, setFocus } ) {
+		const { value, citation } = attributes;
+
+		return (
+			<blockquote className="blocks-pullquote">
+				<Editable
+					value={ value || 'Write quote...' }
```

Comment:
1. Should be translated via `wp.i18n.__`
2. Should use ellipsis character for semantic meaning over three dots (`…`)